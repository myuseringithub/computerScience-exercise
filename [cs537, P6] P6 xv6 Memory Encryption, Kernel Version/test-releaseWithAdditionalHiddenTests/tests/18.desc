[8] A few pages exist, but ptentry array is very large. Ensure that number of pages (beyond dummy) is what we expect them to be