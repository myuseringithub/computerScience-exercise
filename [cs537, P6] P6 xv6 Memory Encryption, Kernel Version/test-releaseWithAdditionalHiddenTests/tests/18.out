XV6_TEST_OUTPUT Passed!
